test/fips/fips_test.go

```diff
@@ -0,0 +1,88 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+//go:build !windows
+
+package acceptance
+
+import (
+	"log"
+	"testing"
+	"time"
+
+	"github.com/aws/amazon-cloudwatch-agent-test/environment"
+	"github.com/aws/amazon-cloudwatch-agent-test/internal/common"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/metric"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/metric/dimension"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/status"
+	"github.com/aws/aws-sdk-go-v2/aws"
+)
+
+const (
+	namespace            = "FIPSTest"
+	agentRuntime         = 3 * time.Minute
+	agentConfigLocalPath = "agent_configs/config.json"
+	agentConfigPath      = "/opt/aws/amazon-cloudwatch-agent/bin/config.json"
+)
+
+var envMetaDataStrings = &(environment.MetaDataStrings{})
+
+func init() {
+	environment.RegisterEnvironmentMetaDataFlags(envMetaDataStrings)
+}
+
+func TestFIPS(t *testing.T) {
```

Comment:
Why are these in the style of the test suite but are not in the metric_value_benchmark test or any newly created test suite?